twig/ION/forms/controller.twig

```diff
@@ -0,0 +1,192 @@
+                                });
+                            }
+                            prevDisabled = false;
+                        });
+                    }
+                });
+            </script>
+        {% endif %}
+        {% if activeFormIndex+1 < formKeys|length and activeFormIndex > -1%}
+            <ion-text slot="end" class="ion-activatable" id="multi_form_next_{{ pid }}">
+                <ion-ripple-effect></ion-ripple-effect>
+                {{ forms[formKeys[activeFormIndex+1]]|raw }}&nbsp;<ion-icon name="arrow-forward"></ion-icon>
+            </ion-text>
+            <script>
+                var nextDisabled = false;
+                $(document).on('click.multi_form_nav_{{ pid }}', "#multi_form_next_{{ pid }}", function() {
+                    if (!nextDisabled) {
+                        nextDisabled = true;
+                        window.ACE.API.POST("{{ formUrl }}", {
+                            'multi_form_nav': 'next'
+                        }).then(function(resp) {
+                                if (resp.json.redirect.url.length > 1) {
+                                window.ACE.API.ContentRetriever(resp.json.redirect.url, {}, "", function() {
+                                    window.ACE.Mulch.doRemovePageByIndex("main_navigator", window.ACE.Mulch.queryByRoute("main_navigator", "{{ route }}"));
+                                });
+                            }
+                            nextDisabled = false;
+                        });
+                    }
+                });
+            </script>
+        {% endif %}
+    </ion-item>
+    <hr>
+{% endif %}
+<form id="{{ formId }}">
+    <ion-list>
+        {% if not twigOverrides.replace %}
+            {% if form is defined and form != '' %}
+                {% include form|replace({'REPLACE':'ION'}) %}
+            {% else %}
+                {% import '@form_config_ION/simple_form.twig' as simpleForm %}
+                {{ simpleForm.render(_context) }}
+            {% endif %}
+        {% else %}
+            {% if includePath %}
+                {% include includePath|replace({'REPLACE':'ION'}) %}
+            {% else %}
+                <script>
+                    $(function() {
+                        window.ACE.Ion.GenerateAlert({
+                            "message": "includePath not defined!"
+                        });
+                    })
+                </script>
+            {% endif %}
+        {% endif %}
+    </ion-list>
+</form>
+<script>
+    window.ACE.__FORMS = {
+        __allowSaveRan: false,
+        addon_changes: {},
+        fields: [],
+        allowSave: function () {
+            if (!window.ACE.__FORMS.__allowSaveRan) {
+                window.ACE.__FORMS.__allowSaveRan = true;
+                $("#save-settings-form-{{ pid }}").removeAttr('disabled');
+            }
+        }
+    };
+
+    $(document).on('click.settings-save-{{ pid }}', '#save-settings-form-{{ pid }}', function() {
```

Comment:
This will undoubtedly be a huge problem.  If you load a form for the second time, you'll be tripping this twice, then three times, etc.  This is continually binding to the same element using a global...